plugin/trino-redshift/pom.xml

```diff
@@ -21,7 +21,7 @@
         <dependency>
             <groupId>com.amazon.redshift</groupId>
             <artifactId>redshift-jdbc42</artifactId>
-            <version>2.1.0.12</version>
+            <version>2.1.0.17</version>
```

Comment:
Coul you run all Redshift tests in a mirror PR? It requires modification of trino-redshift pom.xml  and  ci.yml for extending timeout.

Reply:
Tested internally. All tests have passed for Redshift.

Reply:
I've verified these passed separately. Thanks for reminder.